ons-spark/spark-functions/logistic-regression.md

```diff
@@ -28,14 +28,18 @@ We can read the data in as follows:
 
 from pyspark.sql import SparkSession
 import pyspark.sql.functions as F
+import yaml
 
 import pandas as pd
 
 #local session 
 spark = (SparkSession.builder.master("local[2]").appName("logistic-regression").getOrCreate())
 
+with open("ons-spark/config.yaml") as f:
```

Comment:
That path works for me when I use the notebook file in raw-notebooks/logistic-regression, but not if I try running that code chunk from the markdown directly. I don't know if that is really a problem, but also I notice we tend to specify the path as "ons-spark/config.yaml" for the sparklyr version and then use this relative path for pyspark. 

I've changed it for now to make it consistent with the rest of the book, but I'm just wondering if it makes sense to raise another issue to look at whether it makes the most sense to use relative or absolute filepaths here. It might have already been discussed before though!

Reply:
This file path to config works for R script